### PR TITLE
Help page venueless

### PR DESCRIPTION
**Pull request by blulady**
Created a help page for venueless in the website to have one source of truth and maintain a consistent appearance since the other pages (COC & Sponsors) also come from our website.

Screenshot 2024-09-14 at 08-42-41 Help Page

### CODE REVIEW

**Comment by tim-schilling** [comment]
@blulady What do you think of the following changes?

Screenshot 2024-09-14 at 08-49-18 Help Page
Screenshot 2024-09-14 at 08-49-29 Help Page
Screenshot 2024-09-14 at 08-49-40 Help Page

**Comment by blulady** [comment]
> @blulady What do you think of the following changes?


@tim-schilling I think these are excellent solutions to the problem I was having.

File: src/venueless-help-page.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      </div>
+    </header>
+  </div>
+</div>
+<!-- <div class="bg-gray-100 block-container">
+  <div class="wrapper">
+    <div class="flex items-center justify-center min-h-screen">
+      <div class="gap-8 list-disc lg:gap-16 md:columns-2 prose lg:prose-lg *:break-inside-avoid">
+          <p>General Venuless Questions?</p>
+          <p>Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
```
Comment (drewbrew) [inline src/venueless-help-page.html]: We can't have the link on our repo because then people who don't have tickets can join

**Review by drewbrew — CHANGES_REQUESTED** [review]

File: src/venueless-help-page.html (hunk 1 above)
Comment (tim-schilling) [inline src/venueless-help-page.html]:
```suggestion
          <p>Come Visit Our Help Desk in <a href="https://djangoconus2024.slack.com">Slack </a></p>
```

File: src/venueless-help-page.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <div class="wrapper">
+    <header class="flex flex-wrap items-center gap-8 lg:gap-16">
+      <img
+        src="/assets/img/theme/hero-img.webp"
+        alt="welcome"
+        class="hidden w-full max-w-80 md:block" />
+      <div class="flex-1 max-w-screen-lg">
+        <h1 class="mb-8 pageheading">{{ title }}</h1>
+        <p class="subheading">
+          Need Help? This is the place! Or, the go to <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack</a> Page.
```
Comment (tim-schilling) [inline src/venueless-help-page.html]:
```suggestion
          Need Help? This is the place! Or, the go to <a href="https://djangoconus2024.slack.com">Slack</a> Page.
```

File: src/venueless-help-page.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  </div>
+</div>
+<!-- <div class="bg-gray-100 block-container">
+  <div class="wrapper">
+    <div class="flex items-center justify-center min-h-screen">
+      <div class="gap-8 list-disc lg:gap-16 md:columns-2 prose lg:prose-lg *:break-inside-avoid">
+          <p>General Venuless Questions?</p>
+          <p>Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
+          <p>Technical Issues?</p>
+          <p>Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
```
Comment (tim-schilling) [inline src/venueless-help-page.html]:
```suggestion
          <p>Come Visit Our Help Desk in <a href="https://djangoconus2024.slack.com">Slack </a></p>
```

File: src/venueless-help-page.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        </div>
+    </div>
+  </div>
+</div> -->
+<div class="block-container bg-blue">
+  <div class="wrapper">
+    <div class="grid gap-4 lg:grid-cols-4">
+        <div class="p-4 bg-gray-100">
+          <span class="text-xl font-bold">General Venueless Questions?</span>
+          <p class="">Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
```
Comment (tim-schilling) [inline src/venueless-help-page.html]:
```suggestion
          <p class="">Come Visit Our Help Desk in <a href="https://djangoconus2024.slack.com">Slack </a></p>
```

File: src/venueless-help-page.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        </div>
+        <div class="p-4 bg-gray-200">
+          <span class="text-xl font-bold">Technical Issues?</span>
+          <p>Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
```
Comment (tim-schilling) [inline src/venueless-help-page.html]:
```suggestion
          <p>Come Visit Our Help Desk in <a href="https://djangoconus2024.slack.com">Slack </a></p>
```

File: src/venueless-help-page.html (hunk 1 above)
Comment (blulady) [reply src/venueless-help-page.html]: We are porting this over to an iframe in Venueless, they should already have tickets if they are seeing this page.

File: src/venueless-help-page.html (hunk 5 above)
Comment (blulady) [reply src/venueless-help-page.html]: Will people who aren't already in slack be able to access it?

Comment (drewbrew) [reply src/venueless-help-page.html]: Yes, anyone who has access to GitHub can see this page

Comment (tim-schilling) [reply src/venueless-help-page.html]: @blulady, no they will need to join via the link in the email they receive. However, once they are in the server, this URL will allow them to login with whatever credentials they set up.

Comment (blulady) [reply src/venueless-help-page.html]: @tim-schilling Okay... If they already have a link.

File: src/venueless-help-page.html (hunk 1 above)
Comment (tim-schilling) [reply src/venueless-help-page.html]: @blulady I've deactivated that link (zt-2qh4h5fgg...) You'll need to generate a new one if you want to invite someone to the slack.

**Review by tim-schilling — APPROVED** [review]
Looks good to me.

**Review by drewbrew — APPROVED** [review]
Thanks for fixing the link issue!

**Comment by tim-schilling** [comment]
Thank you @blulady!

**Pull request merged**